apps/marketplace/main.scss

```diff
@@ -70,6 +129,26 @@
   margin-top: 3rem;
 }
 
+.maxHeight1 {
+  max-height: 1rem;
+}
+
+.maxHeight2 {
+  max-height: 2rem;
+}
+
+.maxHeight3 {
+  max-height: 3rem;
+}
+
+.maxHeight4Half {
```

Comment:
add 4 and 5, not half...

Reply:
This is used on the summary preview.  4.5 rem is the sweet spot.  4 and 5 show lines of text that get cut off.